buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -469,6 +470,7 @@ class BeamModulePlugin implements Plugin<Project> {
         google_cloud_datastore_v1_proto_client      : "com.google.cloud.datastore:datastore-v1-proto-client:1.6.3",
         google_cloud_spanner                        : "com.google.cloud:google-cloud-spanner:$google_cloud_spanner_version",
         google_http_client                          : "com.google.http-client:google-http-client:$google_http_clients_version",
+        google_http_client_gson                     : "com.google.http-client:google-http-client-gson:$google_http_clients_version",
```

Comment:
Do you really need google_http_client_gson here? Beam already has google_http_client_jackson2 and it seems CloudHealthcare.Builder accepts Jackson.

Reply:
good question.
I used Gson to be consistent w/ how we do things in the healthcare example docs and a CDAP plugin that was developed for the same purpose.
It does not seem strictly necessary.
I will prepare a commit to remove gson dep and migrate to jackson.